tfx/examples/bert_cola/bert_tokenizer_utils.py

```diff
@@ -0,0 +1,75 @@
+import tensorflow as tf
+import tensorflow_text as text
+
+class Special_Bert_Tokenizer():
+
+    def __init__(self, vocab_dir):
+        self.vocab_dir = vocab_dir
+        self._find_special_token()
+
+    def _find_special_token(self):
+        """Find the special token ID's for [CLS] [PAD] [SEP]"""
+        f = open(self.vocab_dir, 'r')
+        i = 0
+        self.SEP_ID = None
+        self.CLS_ID = None
+        self.PAD_ID = None
+        lines = f.read().split('\n')
+        for line in lines:
+            if line == "[PAD]":
+                self.PAD_ID = tf.constant(i, dtype=tf.int64) 
+            elif line == "[CLS]":
+                self.CLS_ID = tf.constant(i, dtype=tf.int64) 
+            elif line == "[SEP]":
+                self.SEP_ID = tf.constant(i, dtype=tf.int64) 
+            i += 1
+            if self.PAD_ID != None \
+                and self.CLS_ID != None \
+                and self.SEP_ID != None:
+                break
+
+    def tokenize_single_sentence(
+        self,
+        sequence,
+        max_len=128,
```

Comment:
The max_length may be unnecessarily large or too small for Cola. Lets figure out what is appropriate.

Reply:
yes

Reply:
oh and also, this is capped by Bert at 512